### ISSUE TITLE
WebApp creation Issue

### ISSUE THREAD

**Issue by sebojolais** [OpenBoard-org/OpenBoard#837]
Hi,

I follow the tutorial described in the page 6 of the « hint and tips » called « Web Mode » in order to create an application from a website.

I observed two issues in my side. 

Issue 1: a new created application is not visible on the right panel in the applications list.
Steps to reproduce:
- Go to the Web view
- Open this [URL](https://www.micetf.fr/boulierHTML5/)
- Click on the button to create application
- Let the option ‘Whole page’ and click on the ‘Create Application’ button.
- The Board view is shown with the app on the screen.
- But the app is not visible on the right side, in ‘Applications’ → ‘Web’.
- Even though the folder of the new app is well created in this folder: 
`/home/.local/share/OpenBoard/interactive content/Web/`
- Workaround: if I close OpenBoard and reopen it, the new app is visible.

Issue 2: a new created application is never visible on the right panel if I delete it from the Board.
Steps to reproduce:
- Go to the Web view
- Open this [URL](https://www.micetf.fr/boulierHTML5/)
- Click on the button to create application
- Let the option ‘Whole page’ and click on the ‘Create Application’ button.
- The Board view is shown with the app on the screen.
- Delete it on the Board page
- Close OpenBoard
- The folder of the new app is deleted
`/home/.local/share/OpenBoard/interactive content/Web/`
- And if I reopen OpenBoard, the new App is never shown in the right side.

So in short words, if I want to see the new created application in the right panel after his creation, I have to keep this application on the Board page and I have to restart OpenBoard.

Is it the normal behaviour ? Because it does not follow the tutorial of the « hint and tips ».
Thank you in advance for your answer.

System info :
Ubuntu 22.04 (KDE Neon 22.04)
OpenBoard compiled from the dev branch at this last commit 1420d5f36cb9188081b6b9bccaeedcb1f0e97f8e
Qt 5.15.6

**Comment by kaamui** [comment]
Hi, these are known issues internally and should be fixed for 1.7.1, 1.7.2 or 1.7.3.

**Comment by kaamui** [comment]
should be fixed with this commit (1.7.1) => 90132948

Tell me if it works for you !

**Comment by letsfindaway** [comment]
For me there remains an issue related to translations: The widget gets created in a folder called "Web" and after restarting OpenBoard it appears in a folder called "Internet". Note that I also have this with widgets created from the embedded browser.

The German localization translates "Web" to "Internet" in two contexts, while it remains "Web" in a third. I think here are some inconsistencies.

I'm also not sure whether "Web" should be translated in the context of the `UBFeaturesController` at all. At least it is not translated in other usages in that class.

**Comment by kaamui** [comment]
Totally, I saw that during development, it won't work for any language where Web is translated by anything else. 

I though it could for example be an issue for Japanese, but I didn't expect it to be an issue for German translation. I created a ticket yesterday about it, 

I was more considering that it should be translated, as for the example of Chinese, or Japanese, it could be strange to have everything in the language except that folder. That said, I don't remember well, but I think applications names are not translated, so it might not be that disturbing, indeed. 

I'm also not a big fan of the name "Web".

**Comment by sebojolais** [comment]
Hi,
1. Yes, the commit 901329484e708b1a13ffdffae050078a5b5fe97a fixes the 2 issues in my side. Thank you !
2. And the name `Web` does not make sense for me too. Maybe `Customized applications` or `User applications` should be better.
Thank you guys

**Comment by letsfindaway** [comment]
> That said, I don't remember well, but I think applications names are not translated, so it might not be that disturbing, indeed.

- Built-in C++ application names are translated.
- Web widget names are not translated. It could be a future extension to provide the name in multiple languages.
- User created applications get their default name from the web page title, so probably in the right language. Anyway the user can edit the name during creation.

**Comment by kaamui** [comment]
> Hi,
> 
>     1. Yes, the commit [9013294](https://github.com/OpenBoard-org/OpenBoard/commit/901329484e708b1a13ffdffae050078a5b5fe97a) fixes the 2 issues in my side. Thank you !
> 
>     2. And the name `Web` does not make sense for me too. Maybe `Customized applications` or `User applications` should be better.
>        Thank you guys

I'm not even sure it should be separated from other applications. If it had to be separated from them, I would imagine a dedicated folder, at the same level than Applications ans Interactivities.
